tests/support/pkg.py

```diff
@@ -821,14 +837,21 @@ def install_previous(self, downgrade=False):
                 self._install_ssm_service()
 
         elif platform.is_darwin():
+            if relenv and platform.is_aarch64():
```

Comment:
Seems like this could be simplified
```
if platform.is_aarch64():
    arch = "arm64"
else:
    arch = "x86_64"
```

Reply:
Yes, I thought of cleaning up and removing classic stuff, but I wanted to minimize the changes and use the format, used in the existing other code, in the hope to minimize changes and have the tests run cleanly.

Always time to go back and do it better :rofl: .  Planning with https://github.com/saltstack/salt/issues/66672 to go through and clean up all that classic stuff, since pre-3006 is no longer supported.